swap_meet/clothing.py

```diff
@@ -1,10 +1,12 @@
-from hashlib import new
 from swap_meet.item import Item
 
 class Clothing(Item):
     def __init__(self, category="Clothing", condition=0):
-        self.category = category
-        self.condition = condition
+        super().__init__(category, condition)
```

Comment:
Great use of `__super__`! Still consider my previous comment about why we might not want `category` to be an argument on the `Clothing` class.